java/src/jmri/jmrix/loconet/SlotManager.java

```diff
@@ -53,13 +53,13 @@ public class SlotManager extends AbstractProgrammer implements LocoNetListener,
      * Time to wait after programming operation complete on LocoNet
      * before reporting completion and hence starting next operation
      */
-    static public int postProgDelay = 100; // this is public to allow changes via script
+    static public int postProgDelay = 50; // this is public to allow changes via script
 
     public int slotScanInterval = 50; // this is public to allow changes via script and tests
 
-    public int serviceModeReplyDelay = 20;  // this is public to allow changes via script and tests
+    public int serviceModeReplyDelay = 20;  // this is public to allow changes via script and tests. Adjusted by UsbDcs210PlusAdapter
```

Comment:
Changing the comment to the correct variable?
Should the UsbDcs240PlusAdapter "adjust" this too?

Reply:
The UsbDcs240PlusAdapter class changes the service mode time but not the ops mode time. I adjusted the comment here to reflect that.